packages-ts/gauntlet-terra-contracts/src/lib/inspection.ts

```diff
@@ -12,3 +19,23 @@ export const getLatestOCRConfigEvent = async (provider: LCDClient, contract: Acc
 
   return setConfigTx?.logs?.[0].eventsByType['wasm-set_config']
 }
+
+export const getLatestOCRNewTransmissionEvent = async (provider: LCDClient, contract: AccAddress) => {
+  let transmissionTx = (
+    await provider.tx.search({
+      events: [
+        {
+          key: 'wasm-new_transmission.contract_address',
+          value: contract,
+        },
+      ],
+      'pagination.limit': '1',
+      order_by: 'ORDER_BY_DESC',
+    })
+  ).txs
+
+  return transmissionTx.length > 0 ? transmissionTx[0]?.logs?.[0].eventsByType['wasm-new_transmission'] : null
```

Comment:
Will return undefined of the `length` is zero
```suggestion
  return transmissionTx[0]?.logs?.[0].eventsByType['wasm-new_transmission']
```